assets/dev/js/editor/document/commands/base/command-disable-enable.js

```diff
@@ -6,6 +6,10 @@ export default class CommandDisableEnable extends CommandHistory {
 		elementorModules.ForceMethodImplementation();
 	}
 
+	static getTitle() {
```

Comment:
Actually, we don't need this method. We can rely on `getName` when the return value is changed.